### DISCUSSION TITLE
Deployments not triggering

### DISCUSSION THREAD

**Discussion by carlomallone** [vercel/community#2456] [Help]
### Summary

Hey there! I need help with deployments. Every time I push something to GitLab, the `external` pipeline step doesn't work, and Vercel deployments don't get triggered. Also, hooks (via POST or GET) don't trigger any deployments, even if I get a `pending` success response. Any ideas? Removing and re-connecting the repo in the `GIT` settings panel didn't work.

**Comment by pzehle** [comment]
I have this same issue. In my case, when I push, a new pipeline is triggered in my Gitlab instance, however, it pauses itself and it adds the "stuck" label, with the message: "There is no active runner to trigger this pipeline". I have to manually run `vercel deploy`, which removes the purpose of having a CI in the first place.
